src/fetch.js

```diff
@@ -30,6 +30,18 @@ let fetcher = {
             .then(response => response.json())
             .then(data => data)
             .catch(err => console.log('err', err))
+    },
+
+    fetchAllUsers() {
+        const fetchedAllUsers = 'https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers'
```

Comment:
```suggestion
        const allUsers = 'https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers'
```

I think taking `fetched` from the name makes it a little more readable, especially since the function is called `fetchAllUsers`

Reply:
I just saw this below, would this interefere with the allUsers in index.js?
If so, it's fine to leave this as is.

Reply:
I think it might. But I probably should look into renaming this anyways. I was on the fence with `fetchedAllUsers`. Maybe I could `retrievedAllUsers`?